voc/password_based_authentication.go

```diff
@@ -29,3 +29,7 @@ type PasswordBasedAuthentication struct {
 	*Authenticity
 	Activated bool `json:"activated"`
```

Comment:
is there a difference between `activated` an `enabled`

Reply:
No, major inconsistencies in the ontology. The code reflects the ontology.